react/components/atoms/radio/radio.js

```diff
@@ -0,0 +1,183 @@
+import React, { PureComponent } from "react";
+import { Platform, StyleSheet, Text, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles, capitalize } from "../../../util";
+
+import { Touchable } from "../touchable";
+
+export class Radio extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            label: PropTypes.string,
+            value: PropTypes.any,
+            checked: PropTypes.bool,
+            disabled: PropTypes.bool,
+            variant: PropTypes.string,
+            size: PropTypes.number,
+            onUpdateChecked: PropTypes.func,
+            style: ViewPropTypes.style

```

Comment:
Does this respect the usual ordering between event handling + style props?

Reply:
Yap, in the majority of the components style is always at the bottom (in a small number of components its not, but I'll make a PR fixing it).
In `sdk-components-react` there is only one component, and its different: https://github.com/ripe-tech/ripe-sdk-components-react/blob/47bdc3fd91af19ff74ee69783d183a75495238a9/react/components/organisms/ripe-image/ripe-image.js#L253-L257. Do I change it?

Reply:
Yep, go ahead. Thanks for keeping things tidy 🙏